From 4f3cf1970bb90a814b9ba4b1affe50266c2c4b31 Mon Sep 17 00:00:00 2001
From: qi duan <qi.duan@amlogic.com>
Date: Thu, 4 Jan 2024 17:19:44 +0800
Subject: [PATCH] eth: stmmac: stmmac driver move to common_driver for u [1/1]

PD#SWPL-152600

Problem:
stmmac driver move to common_driver

Solution:
stmmac drivr move to commmon_driver

Verify:
ah212

Change-Id: I868fa77b472f8de47fcd452831fb0872e41785b7
Signed-off-by: qi duan <qi.duan@amlogic.com>
---
 drivers/net/ethernet/stmicro/Kconfig  | 2 +-
 drivers/net/ethernet/stmicro/Makefile | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/stmicro/Kconfig b/drivers/net/ethernet/stmicro/Kconfig
index cc136b4c9afd..3b2e7547386d 100644
--- a/drivers/net/ethernet/stmicro/Kconfig
+++ b/drivers/net/ethernet/stmicro/Kconfig
@@ -18,6 +18,6 @@ config NET_VENDOR_STMICRO

 if NET_VENDOR_STMICRO

-source "drivers/net/ethernet/stmicro/stmmac/Kconfig"
+#source "drivers/net/ethernet/stmicro/stmmac/Kconfig"

 endif # NET_VENDOR_STMICRO
diff --git a/drivers/net/ethernet/stmicro/Makefile b/drivers/net/ethernet/stmicro/Makefile
index 72fd1f6ab9b2..2f528fe62ac3 100644
--- a/drivers/net/ethernet/stmicro/Makefile
+++ b/drivers/net/ethernet/stmicro/Makefile
@@ -3,4 +3,4 @@
 # Makefile for the STMicroelectronics device drivers.
 #

-obj-$(CONFIG_STMMAC_ETH) += stmmac/
+#obj-$(CONFIG_STMMAC_ETH) += stmmac/
--
2.34.1

